modules/nextflow/src/main/groovy/nextflow/executor/BridgeExecutor.groovy

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2020, Seqera Labs
+ * Copyright 2013-2019, Centre for Genomic Regulation (CRG)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nextflow.executor
+import java.nio.file.Path
+import java.util.regex.Pattern
+
+import groovy.util.logging.Slf4j
+import nextflow.processor.TaskRun
+/**
+ * Processor for BRIDGE resource manager (DRAFT)
+ *
+ * See https://github.com/cea-hpc/bridge 
+ *
+ *
+ * @author Eric Bonnet <eric.d.bonnet@gmail.com>
+ */
+@Slf4j
+class BridgeExecutor extends AbstractGridExecutor {
```

Comment:
In new classes, we tend to use `@CompileStatic` annotation.

Reply:
Tried to put @CompileStatic but I got this error message during compilation:
/env/export/v_home/q_unix/bonnet/dev/nf/modules/nextflow/src/main/groovy/nextflow/executor/BridgeExecutor.groovy: 65: [Static type checking] - Cannot call <T> java.util.List <String>#leftShift(T) with arguments [long]
 @ line 65, column 13.
               result << '-T' << task.config.getTime().toSeconds()
               ^

Reply:
Add a `toString()` to the seconds value

Reply:
Thanks. Works fine now.